ui/desktop/app/templates/scopes/scope/projects/targets/target.hbs

```diff
@@ -37,7 +37,49 @@
           />
         </Hds::ApplicationState>
       </bc.Body>
-      <bc.Sidebar class='target-details-sidebar'>{{@model.id}}</bc.Sidebar>
+      <bc.Sidebar>
+        <div class='target-details-sidebar'>
+          <div
+            class='title hds-typography-display-200 hds-font-weight-semibold'
+          >
+            {{t 'resources.target.details'}}
+          </div>
+          <Rose::MetadataList @orientation='vertical' as |list|>
+            {{! Target name }}
+            <list.Item @icon='target' @color='faint'>
+              {{@model.name}}
+            </list.Item>
+            {{! Project name }}
+            <list.Item @icon='grid' @color='faint'>
+              {{@model.project.displayName}}
+            </list.Item>
+            {{! Target description }}
+            {{#if @model.description}}
+              <list.Item @icon='message-circle-fill' @color='faint'>
+                {{@model.description}}
+              </list.Item>
+            {{/if}}
+            {{! Target type }}
+            <list.Item class='resource-type' @icon='terminal' @color='faint'>
+              {{#if @model.isSSH}}
+                {{t 'resources.target.types.ssh'}}
+              {{else}}
+                {{t 'resources.target.types.tcp'}}
+              {{/if}}
+            </list.Item>
+            {{! Target ID }}
+            <list.Item @icon='hash' @color='faint'>
+              <Copyable
```

Comment:
@cameronperera 
None blocking comment: You already mentioned on the PR description, but I understand coming PR will change this ad-hoc copyable implementation by the HDS, is that right?

Reply:
Correct. It will be updated in a future PR. I did want to include the functionality still even if we plan to change it soon.